common/document_parser/lib/responsibility_parse/responsibility_parser.py

```diff
@@ -0,0 +1,248 @@
+from common.document_parser.lib.section_parse import add_sections
+import re
+from string import punctuation
+from common.document_parser.lib import entities
+## need "roman"
+from glob import glob
+from tqdm import tqdm
+import json
+from common.document_parser.cli import get_default_logger
+import pandas as pd
+import os
+import pandas as pd
+
+
+class ResponsibilityParser:
```

Comment:
@mishoe There were 2 `ResponsibilityParser` classes uploaded. Was that intentional?
common/document_parser/lib/responsibility_parse/responsibility_parser.py
common/document_parser/lib/section_parse/parse_responsibilties.py